counterpartylib/test/unit_test.py

```diff
@@ -8,6 +8,7 @@
 FIXTURE_SQL_FILE = CURR_DIR + '/fixtures/scenarios/unittest_fixture.sql'
 FIXTURE_DB = tempfile.gettempdir() + '/fixtures.unittest_fixture.db'
 
+@conftest.add_fn_property(DISABLE_ARC4_MOCKING=True)
```

Comment:
arc4 mocking is disabled for the unit_test vectors so they don't all need to be update (which is a hell)